src/neo/Network/P2P/RemoteNode.cs

```diff
@@ -196,6 +197,7 @@ private void OnVerack()
         {
             verack = true;
             system.TaskManager.Tell(new TaskManager.Register { Version = Version });
```

Comment:
reuse the same Register object?

Reply:
I prefer if we remove and merge SyncManager with TaskManager, as @Tommo-L mentioned recently.
What are the advantages of separated ones?